Firestore/Source/Local/FSTLocalDocumentsView.mm

```diff
@@ -159,6 +171,26 @@ - (DocumentMap)documentsMatchingDocumentQuery:(const ResourcePath &)docPath {
   return result;
 }
 
+- (DocumentMap)documentsMatchingCollectionGroupQuery:(FSTQuery *)query {
+  HARD_ASSERT(query.path.empty(),
+              "Currently we only support collection group queries at the root.");
+  std::string collection_id = MakeString(query.collectionGroup);
```

Comment:
Ultranit: I think Objective-C standard uses `camelCase`, while C++ uses `snake_case` for local variables. Since this is chiefly an Objective-C class, consider renaming to `collectionID`.

Reply:
I like this rationale, and I think I'm going to be doing that in the future.